native/src/components/ExportEventButton.tsx

```diff
@@ -79,7 +86,12 @@ const ExportEventButton = ({ event }: ExportEventButtonType): ReactElement => {
     if (editableCalendars.length === 0) {
       showSnackbar({ text: 'noCalendarFound' })
     } else if (editableCalendars.length === 1) {
```

Comment:
```suggestion
    } else if (editableCalendars[0]) {
```

Reply:
I don't understand why that is necessary, can you explain it to me?

Reply:
because its cleaner to check if there is this index `0` instead doing and optional chaining afterwards

Reply:
Learned something new here, first time I see that. Quite annoying though that both checks are needed? Or would only the check `0 in editableCalendars` be enough as well?

Reply:
Yup, I didn't know that either. I don't think only checking `0 in editableCalendars` is enough either because if there is more than one editable calendar, it would also have a 0 :(